youtube_to_anki/main.py

```diff
@@ -32,6 +34,11 @@ def make_package(
     default=None,
     help="Where to export the deck. Defaults to '<deck_name>.apgk'.",
 )
+@click.option(
+    "--resolution",
+    default=360,
+    help="Video resolution (p). Defaults to 360.",
```

Comment:
not sure if this is a sensible default, depends a lot on the user needs. For me 360 is a sweet spot where the image is big enough to make out important details, but the file size & download time are not too high.